qh_stringinthash.h

```diff
@@ -25,8 +25,8 @@
 typedef struct _php_qh_stringinthash_obj php_qh_stringinthash_obj;
 
 struct _php_qh_stringinthash_obj {
-	zend_object   std;
 	qhi          *hash;
+	zend_object   std;
 };
```

Comment:
The PHP 5/PHP 7 supporting version could do something like:

Shared:
```
#if PHP_VERSION_ID < 70000
#define QH_PHP5_STD_OBJ zend_object   std;
#define QH_PHP7_STD_OBJ
#else
#define QH_PHP5_STD_OBJ
#define QH_PHP7_STD_OBJ zend_object   std;
#endif
```

Struct definition:
```
struct _php_qh_stringinthash_obj {
    QH_PHP5_STD_OBJ;
    qhi *hash
    QH_PHP7_STD_OBJ;
};
```

As an example.